third_party/scsi_mock/scsi_mock_module.h

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 Google LLC
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// version 2 as published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+#ifndef NVME2SCSI_SCSI_MOCK_H
+#define NVME2SCSI_SCSI_MOCK_H
+
+#define VERSION "0.1"
+
+static const char kName[] = "SCSI2NVMe SCSI Mock";
+static const int kQueueCount = 1;
```

Comment:
I also wonder what the benefit of having even these in the header is--external code probably doesn't want a symbol called `kName` and we don't have multiple source files to share these between--but constants are much lower risk, at least.